doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -58,7 +58,12 @@ The default values of the VOF parameters are given in the text box below.
 			set output auxiliary fields 	= false
 			set surface tension coefficient = 0.0
 			set phase fraction gradient filter = 0.5
-			set curvature filter 		= 0.5			
+			set curvature filter 		= 0.5	
+            
+            subsection marangoni effect
+                set enable = false
+                set surface tension gradient = 0.0
+            end subsection		
```

Comment:
I think we should fix the indentation here, so that it's clear that the subsection marangoni effect is inside of the surface tension force

Reply:
Agreed

Reply:
Done.